src/styles/saas-list.less

```diff
@@ -1,53 +1,93 @@
+.build-applications-view {
+
+  h1 {
+    color: @saas-text;
+    margin: 0;
+    width: 100%;
+    padding: 20px;
+    padding-left: @grid-gutter-width;
+  }
+}
+
 .saas-offerings-container {
-  background: @color-pf-black-200;
-  padding-left: 0px;
 
   .spinner-container {
     padding-top: 60px;
     padding-bottom: 60px;
   }
 
   .card-view-pf {
-    padding-left: 21px;
-  }
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    flex-direction: column;
+    padding: 0;
+    margin: 0 (@grid-gutter-width / 2);
 
-  .card-view-pf .card-content {
-    background-color: @color-pf-white;
-    cursor: pointer;
-    margin: 0;
+    .card-content {
+      cursor: pointer;
```

Comment:
This is often the sign of an accessibility problem. Not something for you to change @cshinn, but if the cards aren't links, you won't be able to focus them with a keyboard or screenreader. We should look at this as a follow on.